content/en/observability_pipelines/set_up_pipelines/generate_metrics/datadog_agent.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Generate Metrics for the Datadog Agent
+disable_toc: false
+---
+
+## Overview
+
+TKTK
+
+{{% observability_pipelines/use_case_images/generate_metrics %}} 
+
+This document walks you through the following steps:
+1. The [prerequisites](#prerequisites) needed to set up Observability Pipelines
+1. [Setting up Observability Pipelines](#set-up-observability-pipelines)
+1. [Connecting the Datadog Agent to the Observability Pipelines Worker](#connect-the-datadog-agent-to-the-observability-pipelines-worker)
+
+## Prerequisites
+
+{{% observability_pipelines/prerequisites/datadog_agent %}}
+
+## Set up Observability Pipelines
+
+1. Navigate to [Observability Pipelines][1].
+1. Select the **Dual Ship** template to create a new pipeline.
```

Comment:
All mentions of `Dual Ship` template for Generate Metrics docs were fixed in this commit: https://github.com/DataDog/documentation/pull/25475/commits/bdd9fbfe59cf6b2ad5f3dc0d8ac13107827517df